### ISSUE TITLE
ESLint in VS code broken after 1.6.1 upgrade

### ISSUE THREAD

**Issue by SirSKillz** [weiran-zsd/dts-cli#180]
### Current Behavior

Inside of VS code, eslint does not function if you extend the dts-cli as suggested in the [README](https://github.com/weiran-zsd/dts-cli#setting-up-vscode) due to `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` being removed from dependencies and moved into dev dependencies in b90cc74fdfa95becb627170903a41723a40076d7

### Expected behavior

After creating the `.eslintrc.js` file as explained in the README, ESLint should function correctly inside of VS code. While it is common and an easy workaround to install these dependencies inside of our own projects, it should still function without doing so.

### Suggested solution(s)

Add both `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` back to the dependencies of this project

### Your environment

```text
System:
    OS: Windows 10 10.0.19044
    CPU: (24) x64 AMD Ryzen 9 5900X 12-Core Processor
    Memory: 20.26 GB / 31.91 GB
  Binaries:
    Node: 16.17.1 - C:\Program Files\nodejs\node.EXE
    npm: 8.15.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 108.0.5359.125
    Edge: Spartan (44.19041.1266.0), Chromium (108.0.1462.46)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    dts-cli: ^1.6.2 => 1.6.2
    typescript: ^4.9.4 => 4.9.4
```

**Comment by aladdin-add** [comment]
ah, my bad! will release a patch soon, thanks for the report! ❤️

**Comment by aladdin-add** [comment]
fixed in v1.6.3: https://github.com/weiran-zsd/dts-cli/releases/tag/v1.6.3
